v3/examples/window/main.go

```diff
@@ -278,12 +316,6 @@ func main() {
 			w.SetMinSize(200, 200)
 		})
 	})
-	sizeMenu.Add("Set Max Size (600,600)").OnClick(func(ctx *application.Context) {
-		currentWindow(func(w *application.WebviewWindow) {
-			w.SetFullscreenButtonEnabled(false)
-			w.SetMaxSize(600, 600)
-		})
-	})
```

Comment:
Was the removal of "Set/Reset Max Size" example intentional here?

Reply:
No, that was probably an aggressive removal of the full screen button.